style.css

```diff
@@ -14,6 +14,16 @@
   -webkit-text-size-adjust: 100%;
 }
 
+.tooltip {
```

Comment:
Tooltip has an arrow at the bottom
You can try to make one with ::before and ::after using these tricks:
https://css-tricks.com/snippets/css/css-triangle/